__Main__.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<p>Again, useless plot. Replace it with a surface plot.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI-Core-Assignments/project-1-Farhad1353/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Can I use several tiny sub-plots with proper description instead of surface plot?</p>